### PR TITLE
docs: update details to describe tile types

### PR DESCRIPTION
**Pull request by dlimiter**
This PR adds some details around the key differences between the different card types.

[Branch preview](https://dev.cultureamp.design/dlimiter-update-tile-docs/components/tile/)

### CODE REVIEW

**Comment by sebpearce** [comment]
@dlimiter When I go to the meatballs icon on the diff page and go to `View file` I'm seeing some extra asterisks: https://github.com/cultureamp/kaizen-design-system/blob/2bfb0e5547f47b393b2f29dcc9fafa35292e9364/site/docs/components/tile.mdx#anatomy

Might be a difference between how it renders on your end and GitHub's view thing 🤔

**Comment by dlimiter** [comment]
Hmm. That's from content that predates my changes – I think it's some superfluous spacing in the instances of strong formatting. Not interpreted in the Github preview, but picked up fine in the Kaizen branch preview. I've removed them just to be safe.

**Review by sebpearce — APPROVED** [review]

File: site/docs/components/tile.mdx [hunk 1]
```diff
@@ -21,39 +21,54 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
...
+    *   Key information to surface that helps people decide between items
+*   **Information** (optional)
+    *   Object with three parameters: text, primaryAction (optional), secondaryAction (optional)
+    *   Causes Information IconButton to appear in upper right corner. When clicked the backside of Tile is revealed.
+    *   Text params is rendered a paragraph, primaryAction and secondaryAction action rendered as buttons.
+*   **Children** (optional)
+    *   Placed in middle of Tile between footer and title content.
+*   **Supplementary footer** (optional)
+    *   **Call to action default button (required):**
+        *   Consider if the label needs to be different across items, especially for people using screen readers. If the labels are the same, they might also need ARIA tags to indicate which card title the button is labeled by or description it’s described by.
```
Comment (didoesdigital) [inline site/docs/components/tile.mdx:54]: ❤️

File: site/docs/components/tile.mdx [hunk 2]
```diff
@@ -21,39 +21,54 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
...
+
+### InformationTile:
+*   **Title (required)**
+    *   1-3 words
+*   **Metadata**(optional)
+    *   Key information to surface that helps people decide between items
+*   **Information** (optional)
+    *   Object with three parameters: text, primaryAction (optional), secondaryAction (optional)
+    *   Causes Information IconButton to appear in upper right corner. When clicked the backside of Tile is revealed.
+    *   Text params is rendered a paragraph, primaryAction and secondaryAction action rendered as buttons.
```
Comment (didoesdigital) [inline site/docs/components/tile.mdx:66]: Fix up "as a paragraph"?

File: site/docs/components/tile.mdx (hunk 1 above)
Comment (dlimiter) [reply site/docs/components/tile.mdx:54]: Haha that was 100% @ellisjas – all kudo's should be directed to her

File: site/docs/components/tile.mdx (hunk 2 above)
Comment (dlimiter) [reply site/docs/components/tile.mdx]: Fixed in both instances 👍

File: site/docs/components/tile.mdx (hunk 1 above)
Comment (didoesdigital) [reply site/docs/components/tile.mdx:54]: Kudos @ellisjas !

**Pull request merged**